Changelog.md

```diff
@@ -20,89 +65,89 @@ Once that hurdle is crossed, the main emphasis will be there, and
 ## Changes
 
 - Install script with cabal: check there is one ghc in $PATH
-([#1632](https://github.com/haskell/haskell-ide-engine/pull/1551), by @jneira)
+([#1632](https://github.com/haskell/haskell-ide-engine/pull/1632), by @jneira)
```

Comment:
Oh my, did we all miss that?

Reply:
Yes. I was horrified when I saw all the link destinations were the same.  It was a rush job :(

Reply:
Thank you for fixing it! so many PRs :O